Client/mods/deathmatch/logic/luadefs/CLuaWorldDefs.h

```diff
@@ -8,6 +8,10 @@
  *****************************************************************************/
 
 #pragma once
+#include <optional>
+#include <variant>
+#include <utility>
+#include <optional>
```

Comment:
Why did you include these?
I dont see them used in the header itself. So, if thats the case, include them in the `.cpp` file only.
Edit: They aren't used in the cpp either. Perhaps remove them totally? Or does it complain without it?

Reply:
It's just a vestige of CLuaFunctionDefs.h. I forgot to remove this.